services/network_config_manager/src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 server.port=9014
 
 #Microservice url configuration
-microservices.node.service.url=http://localhost:9007/nodes
+microservices.ncm.service.url=http://localhost:9014/nodes
```

Comment:
This is a NCM configuration pointing to its own service url? Usually we use a configuration to store service url of _other_ microservices so that current microservice could reach. Therefore there is no need to store the service's own url.